src/SDK/Management/MgmtV1.php

```diff
@@ -25,14 +116,26 @@ public static function setBaseUrl(string $projectId): void
         self::updatePaths();
     }
 
+    /**
+     * Extracts the region from a given project ID.
+     *
+     * @param string $projectId The project ID to extract the region from.
+     * @return string|null The extracted region or null if not found.
+     */
     private static function extractRegionFromProjectId(string $projectId): ?string
     {
         if (strlen($projectId) >= 32) {
             $region = substr($projectId, 1, 5);
             return !empty($region) ? $region : null;
         }
+        return null;
```

Comment:
Backward compatible?

Reply:
@gshriki it is default mostly semantic https://www.php.net/manual/en/functions.returning-values.php